style-guide/_colors.scss

```diff
@@ -77,16 +77,9 @@ $color_grey_quote:           $palette_grey_text;
 $color_grey_text:            $palette_grey_text;
 
 // My Yoast colors.
-$color_chevron:              $palette_grey_medium_dark;
-$color_round_button:         $palette_green_medium_light;
```

Comment:
On https://github.com/Yoast/my-yoast/pull/90 I've replaced the colors currently used on master. The ones in the pending PRs should be updated on each PR before merging.